packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx

```diff
@@ -2,25 +2,93 @@ import { Notice } from "@cerner/terra-docs";
 import { Badge } from 'terra-folder-tree/package.json?dev-site-package';
 
 import FolderTreePropsTable from 'terra-folder-tree/src/FolderTree?dev-site-props-table';
+import FolderTreeItemPropsTable from 'terra-folder-tree/src/subcomponents/FolderTreeItem/?dev-site-props-table';
 
-import FolderTree from './Examples.2/BasicFolderTree?dev-site-example';
+import BasicFolderTree from './Examples.2/BasicFolderTree?dev-site-example';
 
 <Badge />
 
-# Terra FolderTree
+# Terra Folder Tree
 
 <Notice variant="caution" ariaLevel="2">
 Terra Folder Tree is currently under alpha release. There may be breaking changes between releases until it is stable with the 1.0 release.
```

Comment:
I believe at the completion of this story we should be ready to remove the alpha release designation correct?  If so we will want to remove this notice.

Reply:
Updated: [b4d6f64](https://github.com/cerner/terra-framework/pull/1973/commits/b4d6f64ca6a5e2347f1febb9a55894d19625dd27)